hicommonwealth/commonwealth
# Add Governor Counting Simple Support For Oz Governance

This adds support for communities with GovernorCountingSimple governance contracts, mainly divastaking

## Link to Issue
Closes: #5588

## Description of Changes
This PR effectively adds support for Divastaking and for governor contracts that are of the type `GovernorCountingSimple`. 

## Test Plan
- Migrate the database to ensure new changes
- Navigate to the Divastaking community on localhost 
- Navigate to the proposals page and check if it is loading (http://localhost:8080/divastaking/proposals)
- If it's loading and throws no errors, check a few individual proposal pages
- The description and votes should also load.
- If all of this works, the PR should be good to go
